services/frontend-service/src/ui/Batch.tsx

```diff
@@ -99,6 +100,25 @@ export const ConfirmationDialogProvider = (props: ConfirmationDialogProviderProp
         </IconButton>
     );
 
+    let locksWarning = '';
+    locks?.forEach((lock) => {
+        locksWarning += '- Lock ID: ' + lock[0] + ' | Message: ' + lock[1].message + '\n';
```

Comment:
this is not very "html" like.
I'd rather have an array here and render an array of textfields.
Something like
`locks.map(lock) => <TextField ....`

however, this is a bit nitpicky, I'm also fine with merging it as it is.

Reply:
done!